ci/kuberenetes/distributed.yaml

```diff
@@ -1,10 +1,10 @@
+# Service for nodes RMI-communication 
```

Comment:
Why do we have manifests in the ci/kubernetes dir? We don't have any kubernetes related ci, so probably we should relocate these manifests somewhere to /doc/ dir.